common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -551,7 +529,9 @@
                             }
                         } else {
                             if (state.isTouch) {
-                                this.showTemporaryCaptionMessage(self, "<li>Transcript will be displayed when you start playing the video.</li>"); // jshint ignore: line
+                                self.subtitlesEl.find('.subtitles-menu')
```

Comment:
@cahrens or @andy-armstrong I was unable to get HtmlUtils working in the tests (it's available when previewing locally) so I did some research an opted for this method. It uses `$.text()` then `$.wrap()` so I don't see any problems with escaping. Is this OK?